specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerActiveSecurityAdminRulesList.json

```diff
@@ -0,0 +1,84 @@
+{
+  "title": "Get Azure Virtual Network Manager Active Configuration",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subscriptionA",
+    "resourceGroupName": "myResourceGroup",
+    "networkManagerName": "testNetworkManager",
+    "parameters": {
+      "regions": [
+        "westus"
+      ],
+      "$skipToken": "fakeSkipTokenCode"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "region": "westus",
+            "commitTime": "2020-07-10T18:03:22.2578238+05:30",
+            "id": "/subscriptions/subscriptionA/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/securityAdminConfigurations/myTestSecurityConfig/ruleCollections/testRuleCollection/rules/SampleAdminRule",
+            "configurationDisplayName": "SampleConfig",
```

Comment:
>SampleConfig [](http://example.com/codeflow?start=41&length=12)

I suggest change to SampleSecurityAdminConfig.

Reply:
Any string works here